style.css

```diff
@@ -0,0 +1,210 @@
+@import url('https://fonts.googleapis.com/css2?family=Alef:wght@400;700&family=JetBrains+Mono:ital,wght@0,100..800;1,100..800&display=swap');
+
+* {
+  font-family: JetBrains Mono;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  scroll-behavior: smooth;
+  color: white;
+}
+
+body {
+  height: 100vh;
+  width: 100vw;
+  background-color: #181818;
+}
+
+header {
+  height: 50px;
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  justify-content: center;
+  align-items: center;
+  margin: 25px 5vw;
+}
+
+h1 {
+  text-align: center;
+}
+
+header .logo-container {
+  display: grid;
+  grid-auto-flow: row;
+  font-family:;
+}
+
+header .logo {
```

Comment:
no need to repeat `header .logo` you could just have `.logo` although a better name would be `.header-logo`